backend/data/migrations/20200321002716_create_offers_table.js

```diff
@@ -0,0 +1,35 @@
+module.exports.up = function (knex) {
+  return knex.schema.createTable("offers", (offers) => {
+    offers.increments();
+    offers.boolean("buy_dash").notNullable();
+    offers.string("country").notNullable();
+    offers.string("city").notNullable();
+    offers.string("payment_method").notNullable();
+    offers.string("currency_type").notNullable();
+    offers.string("currency_symbol").notNullable();
+    offers.boolean("dynamic_pricing").notNullable().defaultTo(true);
```

Comment:
all boolean columns should start with `is_`

Reply:
Check